README.md

```diff
@@ -1,11 +1,180 @@
 # CoreFx-TestData
 
-This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
+This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the [dotnet blob feed](https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
```

Comment:
NIT: `binary file` -> `binary files`

Reply:
Also, keeping the feed in parenthesis rather than linking to it might be better. The reason is that the link will take you to the index.json which doesn't really tell you much info, and usually, when following these steps you would want to see the feed you are publishing to in the instructions instead.